src/org/eclipse/swt/accessibility/AccessibleFactory.d

```diff
@@ -95,7 +95,7 @@ class AccessibleFactory {
         }
     }
 
-    private this (int /*long*/ widgetType) {
+    private this (int widgetType) {
```

Comment:
I don't really understand why in some cases `int /*long*/` is replaced with `int` and in some cases the expected `ptrdiff_t`.